labs/coding-102-rest-python-ga/3.md

```diff
@@ -77,20 +77,24 @@ To run this code sample:
     `cd \DevNetCode\~your-name~\coding-skills-sample-code\coding102-REST-python-ga`
 2. Assign the DNA Center IP address or DNS to the **controller** variable.
     * Open the file `get-network-hosts.py`. For example, in Windows, enter: `notepad get-network-hosts.py`
-    * *If you are not using your DNA Center*, use the [DevNet Sandbox](https://developer.cisco.com/site/devnet/sandbox/) DNA Center Lab: [sandboxapic.cisco.com](https://sandboxdnac.cisco.com)
-        * controller='sandboxapic.cisco.com'
+    * *If you are not using your DNA Center*, use the [DevNet Sandbox](https://developer.cisco.com/site/devnet/sandbox/) DNA Center Lab: [sandboxdnac.cisco.com](https://sandboxdnac.cisco.com)
+        * controller='sandboxdnac.cisco.com'
 3. Save the file. If encoding type is an option, select **UTF-8**.
 4. At the command prompt, enter the `python` command followed by the filename.
     * On Windows, enter: `py -3 get-network-hosts.py` or `python get-network-hosts.py`
-    * On Mac OS or Linux, enter: `python3 get-network-hosts.py`
+    * On macOS or Linux, enter: `python3 get-network-hosts.py`
 5. The program should run or display an error message.
 
     You should see a result like the following one.
 
-![](/posts/files/coding-102-rest-python-ga/assets/images/get-hosts.png)
+![](assets/images/get-hosts.png)
 
-## Things to Try
+### Things to try
 * Do you see the ID for each network host?  
 * In the file `get-network-hosts.py` append `?limit=1&offset=1` to the host url so it reads as `url = "https://" + controller + "/api/v1/host?limit=1&offset=1"`. Then, run the code again. What change occurred and why?
```

Comment:
Ohhhh code samples have to change too... or we point to the newer code samples...

Reply:
Okay. We'll discuss with you.

Reply:
I had a chance to talk to Hank on this point also, and the code samples should change in this repo rather than pointing to another repo.